lgsm/functions/alert_rocketchat.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
-# LinuxGSM alert_rocketchat.sh function
-# Author: Alasdair Haig
+# LinuxGSM alert_rocketchat.sh module
+# Author: Daniel Gibbs
```

Comment:
This is probably not your intent, but they way you did this makes it look like you are claiming the merit of other devs.

Reply:
Quite the opposite I made the change to more fairly highlight contributors. 

I am the creator/lead developer of LinuxGSM so my name goes on every file as Author. There was a contributor line underneath that was used for contributors. Sometimes people got mixed up and used the Author instead of contributor.

So instead I decided to rely on, GitHub shows every person who has contributed to the code and does it automatically. 

What normally happened was the first person who created the module put their name in and subsequent contributors don't normally add their name. I felt it was unfair to other contributors as over the years some have contributed more code to a module than the original creator. Or some modules are simply a few lines of code with the person putting their name in and never contributing again. So to make things fair I instead linked to the GitHub contributors page that lists all contributors to LinuxGSM. It is also possible to see contributors to a specific module or using blame to see who wrote specific code within GitHub.

Basically, its to make sure contributors are displayed fairly and automatically. 

You are right to raise this concern and I hope this answers your question 😃

Reply:
I kind of understand what you are saying, but these lines clearly indicate that you are the author of the alert_rocketchat.sh module, which it looks like you aren't. That's why I raised concern.